crates/universal-wallet/fuzz/src/lib.rs

```diff
@@ -50,13 +50,21 @@ impl Arbitrary<'_> for ArbitrarySafeString {
     }
 }
 
+fn bech_prefix() -> &'static str {
+    "test"
+}
+
 #[derive(
     Debug, BorshSerialize, BorshDeserialize, Serialize, Deserialize, UniversalWallet, Arbitrary,
 )]
 pub enum ByteVecInput {
     Hex(#[sov_wallet(display(hex))] Vec<u8>),
-    Base58(#[sov_wallet(display(base58))] Vec<u8>),
+    Base58 {
+        #[sov_wallet(display(base58))]
```

Comment:
seems like a bug in macro, no? Why it does not work with tuple and works with struct?